pages/linux/isoinfo.md

```diff
@@ -0,0 +1,15 @@
+# isoinfo
+
+> Utility programs for dumping and verifying iso images.
+
+- List all the files packed into an iso:
+
+`isoinfo -f -i {{path/to/image.iso}}`
+
+- Extract a specific file to stdout:
```

Comment:
Yeah, that might work. "print" implies text though, and this command supports binary files.